django/core/management/commands/test.py

```diff
@@ -50,6 +51,9 @@ def handle(self, *test_labels, **options):
         TestRunner = get_runner(settings, options['testrunner'])
 
         time_keeper = TimeKeeper() if options.get('timing', False) else NullTimeKeeper()
+        parallel = options.get('parallel')
+        if parallel == 'auto':
+            options['parallel'] = get_max_test_processes(parallel)
```

Comment:
I restored the previous behavior where `DJANGO_TEST_PROCESSES` and a method used by `multiprocessing` to start subprocesses were only consulted only with `--parallel` without value (default). It will now match to `--parallel` and `--parallel=auto`.

Reply:
I just noticed that you don't ever seem to need `get_max_test_processes()`'s argument. Here you do pass `parallel`, but you're only ever passing the default argument value. So it looks like you don't actually need to accept a value.

Reply:
Great catch :dart: